application/console/migrations/m171211_130001_add_several_mfa_email_log_types.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m171211_130001_add_several_mfa_email_log_types
+ */
+class m171211_130001_add_several_mfa_email_log_types extends Migration
+{
+
+    public function safeUp()
+    {
+        $this->alterColumn(
+            '{{email_log}}',
+            'message_type',
+            "enum('invite','welcome','mfa-rate-limit','password-changed','get-backup-codes','refresh-backup-codes'," .
+                "'lost-security-key','mfa-required','mfa-option-added','mfa-option-removed'," .
```

Comment:
Was `mfa-required` supposed to be in this list? I don't see any other uses of that email-log type.

Reply:
Removed.  (There had been another type of email to send. But, it probably won't work here.)